chronix-server-ingestion-handler/src/main/java/de/qaware/chronix/solr/ingestion/format/opentsdb/TsdbMetric.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016 QAware GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package de.qaware.chronix.solr.ingestion.format.opentsdb;
+
+import java.util.HashMap;
+
+/**
+ * A DTO of a TSDB metric. Uses for GSON deserializing.
+ */
+public class TsdbMetric {
+    private String metric;
+    private long timestamp;
+    private double value;
+    private HashMap<String, String> tags;
```

Comment:
\[Checkstyle\] ERROR: Declaring variables, return values or parameters of type 'HashMap' is not allowed\.